actions/apidiff-go/action.yml

```diff
@@ -0,0 +1,44 @@
+name: apidiff-go
+description:
+  "Uses apidiff CLI to analyze pull requests for breaking changes in a Go
+  library's exported symbols."
+
+runs:
+  using: "node20"
+  main: "dist/index.js"
+
+inputs:
+  # path inputs
+  directory:
+    description: "Directory containing the Go repository"
+    required: false
+    default: "./"
+  go-mod-path:
+    description: "Relative path to the go.mod file from the repository root"
+    required: false
+    default: "."
```

Comment:
`go-mod-paths` - support pointing to multiple go mods with multiline yaml

or infer from glob